src/main/java/StrategyPatternClasses/TableGenerator.java

```diff
@@ -0,0 +1,94 @@
+package StrategyPatternClasses;
+
+import automatabuilder.Symbol;
+import interfaces.IAlphabet;
+import interfaces.IState;
+import interfaces.ITransition;
+
+import java.util.List;
+
+/**
+ * Created by oliv on 5/23/15.
+ */
+class TableGenerator {
+    StringBuilder sb            = new StringBuilder();
+    StringBuilder whitespace    = new StringBuilder();
+    String whitespaceString;
+
+    public String getTable(List<IState> stateList, IAlphabet alphabet){
```

Comment:
Well, if we need some graphical representation we need to rewrite/extend TableGenerator anyway. Maybe instead of supplying stream to constructor, just supply to a function?

```Java
public void printTable(PrintStream stream, ...) {
    stream.print(getTable(...));
}
```